src/View/ProgressWindow.py

```diff
@@ -51,6 +56,16 @@ def start(self, funct):
         self.threadpool.start(worker)
         self.exec_()
 
+    def start_load_moving_image(self, selected_files):
+        image_loader = ImageLoader("moving", selected_files, None, self)
+        worker = Worker(image_loader.load,
+                        self.interrupt_flag, progress_callback=True)
+        worker.signals.result.connect(self.on_finish)
+        worker.signals.error.connect(self.on_error)
+        worker.signals.progress.connect(self.update_progress)
+
+        self.threadpool.start(worker)
+
```

Comment:
This progress window class is a base class and inherited by OpenPatientProgressWindow class. This function is specific to image fusion. Hence, this should be moved to another class that inherits ProgressWindow.py instead of putting in in the base class. 
If the class is small and just contains this function then I suggest creating it inside ImageFusionWindow.py